powerbi-docs/connect-data/service-template-apps-create.md

```diff
@@ -38,6 +38,9 @@ To create a template app you can distribute to other Power BI tenants, you need
    >[!Important]
    > The capacity that the app builder workspace is assigned to does not determine the capacity assignment of workspaces where app installers install the app. This means that an app developed in a premium capacity workspace will not necessarily be installed on a premium capacity workspace. Therefore it is **not** recommended to use premium capacity for the builder workspace, as installer workspaces might not be premium capacity, and functionality that relies on premium capacity won't work unless the installer manually reassigns the installed workspace to premium capacity.
 
+   >[!Important]
+   > Enabling the feature switch to develop template apps in the workspace settings, is an irreversible operation.  
```

Comment:
I'd explain it like that:
Enabling the "Develop template app" switch can only be done when the app builder workspace is created and cannot changed later from workspace settings.